### PR TITLE
ci: change helm charts

### PR DESCRIPTION
**Pull request by mcruzdev**
# Changes

- :broom: Update helm chart

After the charts update, I got this result regarding packaging and installation:

image

### CODE REVIEW

File: helm/fmtok8s-email-rest/values.yaml [hunk 1]
```diff
@@ -3,54 +3,22 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: salaboy/fmtok8s-email-service
+  repository: matheuscruzdev/fmtok8s-email-service
```
Comment (salaboy) [inline helm/fmtok8s-email-rest/values.yaml]: Let's change this to `ghcr.io/salaboy/fmtok8s-email-service` right? We need the one hosted in github packages

Comment (salaboy) [reply helm/fmtok8s-email-rest/values.yaml]: @mcruzdev I think we need: `ghcr.io/` because this is going to go to docker hub if we don't add the registry

**Comment by salaboy** [comment]
@mcruzdev I think you also need to change the chart name (directory name) to match the chart name. Can you please do that so I can merge this PR? I will do the changes in c4p so we have all the projects aligned before I move to the Frontend

**Comment by mcruzdev** [comment]
@salaboy 
Done!

**Comment by salaboy** [comment]
@mcruzdev I've notice that this chart in the values.yaml file doesn't have the native image as default.

**Pull request merged**